ddtrace/bootstrap/sitecustomize.py

```diff
@@ -2,37 +2,26 @@
 Bootstrapping code that is run when using the `ddtrace-run` Python entrypoint
 Add all monkey-patching that needs to run by default here
 """
-import logging
-import os
 import sys
-from typing import Any
-from typing import Dict
 
 
-# Perform gevent patching as early as possible in the application before
-# importing more of the library internals.
-if os.environ.get("DD_GEVENT_PATCH_ALL", "false").lower() in ("true", "1"):
-    import gevent.monkey
-
-    gevent.monkey.patch_all()
+LOADED_MODULES = frozenset(sys.modules.keys())
```

Comment:
Good point. Although I think `ALREADY_LOADED_MODULES` or `MODULES_LOADED_AT_STARTUP` might be more specific, given that we are in the `sitecustomize` script here and it might be unclear at which stage of the sitecustomize script these modules were loaded? 🤔